packages/react/src/auto/AutoFormActionValidators.ts

```diff
@@ -8,7 +8,7 @@ export const validateNonBulkAction = (action: ActionFunction<any, any, any, any,
 };
 
 const GadgetApiTriggerSpecId = "gadget/trigger/graphql_api";
-const ApiTriggerRequiredError = `Actions must have API triggers to be used in AutoForms`;
+export const ApiTriggerRequiredError = `"action" must be a valid Gadget action with an API trigger to be used in AutoForms`;
```

Comment:
`ApiTriggerRequiredError` sounds like an actual error class, but instead is an error message.
How about `ApiTriggerRequiredErrorMessage`?

Reply:
Renamed to `TriggerableActionRequiredErrorMessage`